pkg/api/handlers/compat/containers.go

```diff
@@ -463,9 +463,20 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*container.InspectResp
 		state.Running = true
 	}
 
-	// Dockers created state is our configured state
-	if state.Status == define.ContainerStateCreated.String() {
-		state.Status = define.ContainerStateConfigured.String()
+	// map our statuses to Docker's statuses
+	switch state.Status {
+	case define.ContainerStateConfigured.String(), define.ContainerStateCreated.String():
+		state.Status = "created"
+	case define.ContainerStateRunning.String(), define.ContainerStateStopping.String():
+		state.Status = "running"
+	case define.ContainerStatePaused.String():
+		state.Status = "paused"
+	case define.ContainerStateRemoving.String():
+		state.Status = "removing"
+	case define.ContainerStateStopped.String(), define.ContainerStateExited.String():
+		state.Status = "exited"
+	default:
+		state.Status = "" // unknown state
```

Comment:
Also wasn't sure about the "Unknown" state which doesn't map to anything logical in docker's list.

I went with this because that is a variant that the Rust library I'm using expects: https://docs.rs/bollard-stubs/1.47.1-rc.27.3.1/src/bollard_stubs/models.rs.html#1229-1230